.circleci/config.yml

```diff
@@ -60,15 +60,15 @@ jobs:
   build-simulator-dockerfile:
     docker:
       - image: cimg/base:2022.06
-    parallelism: 5
+    parallelism: 6
```

Comment:
It's probably worth leaving a comment as to why the increased parallelism was necessary